docs/examples.html

```diff
@@ -1,127 +1,127 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

```

Comment:
Did you just change line endings here? Why does that have anything to do with CommonJS support?